src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +29,32 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${todo?.id}`}
```

Comment:
maybe be will be better to destructure todo not to write it all the time)

Reply:
my todo has type `todo | null` I don't understand how can I destructured it((